pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/fullcsaugkehubfeaturemembership/create.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  name: gkehubfeaturemembership-full-csau-${uniqueId}
+spec:
+  projectRef:
+    name: project-${uniqueId}
+  location: global
+  membershipLocation: us-south1
+  membershipRef:
+    name: gkehubmembership-full-csau-${uniqueId}
+  featureRef:
+    name: gkehubfeature-full-csau-${uniqueId}
+  configmanagement:
+    configSync:
+      sourceFormat: hierarchy
+      git:
+        gcpServiceAccountRef:
+          name: sa-${uniqueId} 
+        httpsProxy: "https://github.com"
+        syncRepo: "https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit"
+        syncBranch: "master"
+        policyDir: "config-connector"
+        syncWaitSecs: "20"
+        syncRev: "HEAD"
+        secretType: "none"
+    version: "1.18.1"
```

Comment:
Could you make the test case be updating from one management mode (e.g. automatic) to another management mode (e.g. manual)? We want to test the update of this field as well.